src/main/java/seedu/address/model/person/Person.java

```diff
@@ -30,6 +30,7 @@ public class Person {
     private final Profession profession;
     private final Income income;
     private final Details details;
+    private Set<Interaction> interactions = new HashSet<>();
```

Comment:
can be final, too coz the set object itself will not be altered

Reply:
Will change!